xla/debug_options_flags.cc

```diff
@@ -1706,6 +1706,11 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       "xla_use_shardonnay",
       bool_setter_for(&DebugOptions::set_xla_use_shardonnay),
       debug_options->xla_use_shardonnay(), "Whether to use Shardonnay."));
+  flag_list->push_back(
+      tsl::Flag("xla_gpu_kernel_cache_file",
+                string_setter_for(&DebugOptions::set_xla_gpu_kernel_cache_file),
+                debug_options->xla_gpu_kernel_cache_file(),
+                "Path to a file to cache compiled kernels."));
```

Comment:
Maybe explain when do we write and when do we read this file. Maybe something like this?
"If the file doesn't exist then we will write the compilation cache of the first compiled HLO into it.
Once the file exists, the further HLO compilations will read it to reuse the kernels, but not write it."

(see related comment later)

Reply:
done